src/nvim/ex_docmd.c

```diff
@@ -8848,12 +8848,11 @@ makeopens (
   tab_firstwin = firstwin;      /* first window in tab page "tabnr" */
   tab_topframe = topframe;
   for (tabnr = 1;; ++tabnr) {
```

Comment:
Linter wants `tabnr++` instead of `++tabnr` here. (I know you didn't change it, but the linter looks at surrounding lines as well.)